test/new-e2e/tests/agent-metric-logs/log-agent/utils/file_tailing_utils.go

```diff
@@ -3,28 +3,45 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-package logagent
+// utils provides shared common functions so different E2E tests suites can use them.
+package utils
 
 import (
 	"encoding/json"
 	"fmt"
 	"strings"
+	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
+
 	"github.com/DataDog/datadog-agent/test/fakeintake/aggregator"
 	fi "github.com/DataDog/datadog-agent/test/fakeintake/client"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e"
+
 	componentos "github.com/DataDog/test-infra-definitions/components/os"
-	"github.com/stretchr/testify/assert"
 )
 
-// appendLog appen log with 'content', which is then repeated 'reccurrence' times and verifies log contents.
-func appendLog(s *LinuxFakeintakeSuite, content string, recurrence int) {
+type TestHelper interface {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Rather than `TestHelper` this could be a `LogsTestSuite` interface